projects/templates/projects/view_project.html

```diff
@@ -149,6 +149,36 @@ <h4>Current Users</h4>
   {% endfor %}
 </ul>
 
+{% if is_pi %}
+  <h4>Pending Invitations</h4>
+
+  <form class="form-inline" role="form" method="post">
+    {% csrf_token %}
+    {% bootstrap_form invite_form %}
+    <div class="form-group">
+      <button type="submit" class="btn btn-default" name="invite_user">Invite user</button>
+    </div>
+  </form>
+
+  <ul class="list-unstyled">
+    {% for i in invitations %}
+    <li>
+      {{i.email_address}}
```

Comment:
style nit: always include spaces around curlys

```suggestion
      {{ i.email_address }}
```